package.json

```diff
@@ -18,6 +18,7 @@
     "/bin",
     "/dist",
     "/templates",
+    "/rollup-plugin-optimize-lodash-imports",
```

Comment:
feels like a good opportunity for placing it under the old-fashioned `vendor` directory.
What do you think? This way, we can place more packages if needed, and one doesn't need to remember about adding them to `package.json`

Reply:
is vendor included by default or something?